NEWS.md

```diff
@@ -17,6 +17,10 @@ for human readability.
   and [ECOS.jl](https://github.com/jump-dev/ECOS.jl) ([#2147])
 - Passive tracers for arbitrary equations with density and flow variables ([#2364])
 
+#### Changed
+- The (2D) aerodynamic coefficients 
+  `DragCoefficientPressure, LiftCoefficientPressure, DragCoefficientShearStress, LiftCoefficientShearStress` have been relabeled to 
+  `DragCoefficientPressure2D, LiftCoefficientPressure2D, DragCoefficientShearStress2D, LiftCoefficientShearStress2D`. ([#2375])
```

Comment:
This is breaking, so it cannot be done in the v0.11 lifecycle. However, you could deprecate them accordingly so that we can go ahead with this PR without a breaking change.

Reply:
Ah yeah I remember that goes in a different section, right!

Reply:
Hm so this is not super urgent - but we have a breaking change waiting anyways, right? 

https://github.com/trixi-framework/Trixi.jl/pull/2357